docs/translations/zh/developers/index.md

```diff
@@ -1,11 +1,15 @@
 ---
-title: 开发者
+title: Ethereum for Develop
```

Comment:
This file is also incorrect, this is the frontmatter content I see after downloading the file:
```
---
title: 面向开发者的以太坊资源
meta:
  - 
    name: 说明
    content: 本页为以太坊开发者提供入门指南、学习资源以及开发工具。
  - 
    property: og:标题
    content: 面向开发者的以太坊资源
  - 
    property: og:说明
    content: 本页为以太坊开发者提供入门指南、学习资源以及开发工具。
lang: zh-CN
sidebar: 自动
sidebarDepth: 1
---
```
Please ensure all files have been correctly downloaded.